BioArchLinux/minimap2/PKGBUILD

```diff
@@ -3,23 +3,46 @@
 
 pkgname=minimap2
 pkgver=2.26
-pkgrel=1
+pkgrel=2
 pkgdesc='A versatile pairwise aligner for genomic and spliced nucleotide sequences. https://doi.org/10.1093/bioinformatics/bty191'
 arch=('x86_64')
 url="https://github.com/lh3/${pkgname}"
 license=('MIT')
-depends=(zlib)
-provides=()
+depends=('zlib' 'glibc')
+provides=('minimap2-lite' 'sdust')
 source=("${pkgver}.tar.gz::$url/archive/refs/tags/v$pkgver.tar.gz")
 sha256sums=('f4c8c3459c7b87e9de6cbed7de019b48d9337c2e46b87ba81b9f72d889420b3c')
+
+prepare() {
+  cd "${pkgname}-${pkgver}"
+  # Add hardening flags for binaries
+  sed -i '1s/CFLAGS=/CFLAGS+=/' Makefile
+  sed -i '2s/CPPFLAGS=/CPPFLAGS+=/' Makefile
+  sed -i '52s/$(LIBS)/$(LIBS) $(LDFLAGS)/' Makefile
+  sed -i '55s/$(LIBS)/$(LIBS) $(LDFLAGS)/' Makefile
+  sed -i '61s/-lz/-lz $(LDFLAGS)/' Makefile
+  sed -i '57a\\t\ \trm -f "$@"' Makefile
+  sed -i '59s/-csru/csrD/' Makefile
```

Comment:
Using a patch would be more robust than `sed`, because the line numbers can change easily if upstream adds more lines to the top of the Makefile.

Also you could change `libminimap2` to be a shared library instead of static. That would require changing the `.a` file extension to `.so` and changing the build command to

```make
libminimap2.so: $(OBJS)
		$(CC) -shared $(LDFLAGS) -o $@ $^ $(LIBS)
```
I have not tested this though.

Reply:
Done